src/implementations/twig/components/message/message.html.twig

```diff
@@ -52,19 +52,19 @@
     icon: _icon,
     extra_classes: 'ecl-message__icon',
   } only %}
-  {% if _close.icon is defined and _close.icon is not empty %}
-    {% set _icon = _close.icon|merge({ name: 'close-filled' }) %}
-  {% endif %}
+  
   <div class="ecl-message__content">
-    {% include '@ecl/button/button.html.twig' with _close|merge({
-      type: 'button',
-      variant: 'ghost',
-      icon: _icon,
-      extra_classes: 'ecl-message__close',
-      extra_attributes: [{
-        name: 'data-ecl-message-close',
-      }],
-    }) only %}
+    {% if _close is not empty %}
+      {% include '@ecl/button/button.html.twig' with _close|merge({
+        type: 'button',
+        variant: 'ghost',
+        icon: _close.icon|merge({ name: 'close-filled' })|default({}),
```

Comment:
this line seems problematic to me:
the default should probably be set for the _close.icon, 
`_close.icon|default({})|merge({ name: 'close-filled' }),`
which we don't know whether is set, but in any case passing an icon object without checking for the icon_path is not ensuring that the icon will be rendered.